packages/adapter-vercel/index.js

```diff
@@ -340,10 +340,19 @@ async function v3(builder, external, edge, split) {
 				id: route.pattern.toString(), // TODO is `id` necessary?
 				filter: (other) => route.pattern.toString() === other.pattern.toString(),
 				complete: async (entry) => {
-					const src = `${route.pattern
+					let slicedPattern = route.pattern
```

Comment:
we use `snake_case` for variable names

```suggestion
					let sliced_pattern = route.pattern
```